src/profile-logic/profile-data.js

```diff
@@ -207,7 +198,15 @@ export function getSamplesSelectedStates(
       return 'FILTERED_OUT';
     }
 
-    // Walk the call nodes toward the root, and get the call node at the the depth
+    // When there's no selected call node, we don't want to shadow everything
+    // because everything is unselected. So let's decide this is as if
+    // everything is selected so that anything not filtered out will be nicely
+    // visible.
+    if (selectedCallNodeIndex === null) {
+      return 'SELECTED';
+    }
```

Comment:
This is the main change: when there's no selected call node, we decide all call nodes that aren't filtered out will get the "selected" style.

Reply:
Ah, this works!